mender-deb-package

```diff
@@ -0,0 +1,107 @@
+#/bin/bash
+
+set -ex
+
+show_help_and_exit() {
+  cat << EOF
+
+Usage: $0 mender-version
+
+EOF
+  exit 1
+}
+
+version=""
+
+while (( "$#" )); do
+  case "$1" in
+    -*)
+      echo "Error: unsupported option $1"
+      show_help_and_exit_error
+      ;;
+    *)
+      version="$1"
+      shift
+      ;;
+  esac
+done
+
+if [ -z "$version"]; then
+	show_help_and_exit
+fi
+
+#Print misc env variables for debug
+echo "WORKDIR="`pwd`
+echo "PATH="$PATH
+echo "CC="$CC
+echo "GOPATH="$GOPATH
+echo "version=$version"
+
+mkdir debian && mkdir debian/source
+
+#TODO: find a way to take this from the Changelog itself
+dch --create -v ${version}-1 --package mender "Release ${version}. See online docs for Changelog"
+
+echo "10" > debian/compat
+
+echo "3.0 (quilt)" > debian/source/format
+
+cat << EOF > debian/control
+Source: mender
+Maintainer: Mender Team <menderall@northern.tech>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender
+Architecture: armhf
+Depends: \${shlibs:Depends}, \${misc:Depends}
+Description: mender client
+ Mender is an open source over-the-air (OTA) software updater for embedded Linux devices.
+EOF
+
+cp LICENSE debian/copyright
+
+cat << EOF > debian/rules
+#!/usr/bin/make -f
+
+%:
+	dh \$@
+
+override_dh_auto_build:
+	env CGO_CFLAGS="-I/usr/local/include" CGO_LDFLAGS="-L/usr/local/lib" \
+  GOOS=linux GOARCH=arm GOARM=5 CGO_ENABLED=1 \
+  make build
+
+override_dh_auto_install:
+	sed -i 's/prefix ?=/prefix="\$\${DESTDIR}"/' Makefile
```

Comment:
@kacf This is because inside mender client Makefile we use `prefix` only.

I couldn't find a lot of documentation, but it seems like debmake uses `DESTDIR` instead of `prefix` to pint to the destination directory. See [this link](https://www.debian.org/doc/manuals/debmake-doc/ch08.en.html#makefile-sh)

So we could leave this hack or modify the Makefile (so far I assume only the new integration tests are using it)

Reply:
I think we should definitely modify the Makefile. Just add

```
DESTDIR ?= /
prefix ?= $(DESTDIR)
```

This way the user can override either one.

Reply:
Good suggestion :+1: